tests/sdp-negotiation.rs

```diff
@@ -0,0 +1,184 @@
+mod common;
+use common::init_log;
+use str0m::format::Codec;
+use str0m::format::CodecSpec;
+use str0m::format::FormatParams;
+use str0m::format::PayloadParams;
+use str0m::media::Direction;
+use str0m::media::MediaKind;
+use str0m::Rtc;
+use tracing::info_span;
+
+#[test]
+pub fn change_pt_sendrecv() {
+    init_log();
+
+    let (l, r) = negotiate(
+        //
+        &[opus(100, 2)],
+        &[opus(102, 2)],
+        MediaKind::Audio,
+        Direction::SendRecv,
+    );
+
+    // For sendrecv, both sides state their mandatory receive parameters. I.e.
+    // L -> R must be 102
+    // R -> L must be 100
+
+    assert_eq!(
+        &[opus(102, 2)],
+        &**l.codec_config_tx(),
+        "L sends to R on PT 102"
+    );
+    assert_eq!(
+        &[opus(100, 2)],
+        &**l.codec_config_rx(),
+        "L receives from R on PT 100"
+    );
+    assert_eq!(
+        &[opus(102, 2)],
+        &**r.codec_config_rx(),
```

Comment:
Should say `&**r.codec_config_tx()` and the value above should be 100 i.e. R should respect the value that was present in the offer(100)

Reply:
Yeah. Sigh. Maybe I should remove the text. Too many things that can go wrong when rigging the tests.

Reply:
Could write a helper to reduce the duplication to

Reply:
Done